shims/spark311/src/main/scala/com/nvidia/spark/rapids/shims/spark311/Spark311Shims.scala

```diff
@@ -45,6 +45,21 @@ import org.apache.spark.sql.rapids.shims.spark311._
 import org.apache.spark.sql.types._
 import org.apache.spark.storage.{BlockId, BlockManagerId}
 
+abstract class OffsetWindowFunctionMeta2[INPUT <: OffsetWindowFunction] (
```

Comment:
This is never used. Can it be deleted?

Reply:
Yes, of course. Apologies, this is sloppy.